src/GraphQL/TypeExtensions.cs

```diff
@@ -41,6 +41,30 @@ public static bool IsConcrete(this Type type)
         public static bool IsGraphType(this Type type)
             => typeof(IGraphType).IsAssignableFrom(type);
 
+        /// <summary>
+        /// Determines if the specified type represents a named graph type (not a wrapper type such as <see cref="ListGraphType"/>).
+        /// </summary>
+        internal static bool IsNamedGraphType(this Type type)
```

Comment:
See `GraphQLExtensions.GetNamedType(this Type type)` and whole `GraphQLExtensions` class. I propose to merge two classes into one. I see no reason to keep 2 classes for methods that both work with graph types (`GraphQLType` from parser, `IGraphType` or just `Type`).

Reply:
Also I propose to rename `IsNamedGraphType` to `IsNamedType` or rename `GetNamedType` to `GetNamedGraphType`.

Reply:
I’m okay shifting things around a bit. I do like having them separate in the codebase so if we do merge them then I would keep them as separate files defined as partial classes.

Reply:
IsNamedGraphType: just let me know what you prefer. I had IsNamedType but as an extension on Type I figured it would be better if it was named something more specific. And it would better match IsGraphType. 

Unless you have a comment, if GetNamedType isn’t public, I’ll rename it, otherwise I’ll rename this method as you suggested.

Reply:
OK for IsNamedType